qa/tasks/cephfs/test_mds_metrics.py

```diff
@@ -457,7 +463,7 @@ def test_perf_stats_stale_metrics(self):
             log.debug(f'metrics={metrics_new}')
             self.assertTrue(valid)
 
-            global_metrics = metrics_new['global_metrics']
+            global_metrics = metrics_new['global_metrics'][TestMDSMetrics.TEST_FS_NAME]
```

Comment:
Why do you want a new `TEST_FS_NAME` in this PR? Could this just be `self.fs.name`, which is also `cephfs`? Anything else you are trying to fix?
(When multiple fs is enabled, you are mentioning the fsname anyway)

Reply:
Yup, you are correct!! Actually at that time, I just thought to have new variable to fix the failure.

Reply:
Can you fix it then? I've noticed this thing while I was working on another patch.

Reply:
Done!! `TestMDSMetrics.TEST_FS_NAME -> self.fs.name`